bin/setup_dev_machine

```diff
@@ -6,6 +6,9 @@
 # some arguments don't have a corresponding value to go with it such
 # as in the --default example).
 # note: if this is set to > 0 the /etc/hosts part is not recognized ( may be a bug )
+
+set -xe
```

Comment:
please do it separately the same way as here https://github.com/shadow-robot/sr-build-tools/blob/master/ansible/deploy.sh#L3-L4